edgeless_inabox/src/bin/edgeless_inabox.rs

```diff
@@ -122,7 +122,9 @@ fn generate_configs(number_of_nodes: i32) -> Result<InABoxConfig, String> {
         node_confs.push(EdgelessNodeSettings {
             node_id: *node_id,
             agent_url: node_orc_agent_urls.get(node_id).expect("").clone(), // we are sure that it is there
```

Comment:
Guess this is now a "listen" url? Afaik @lukasz-zet wanted to just go for Ports here at some point.

Reply:
Yes, I kept the old names to minimise the changes:

- agent_url/invocation_url -> URLs with the address bound and listening port
- same with _announced -> URLs announced to the ε-ORC, used only during the registration phase